lib/taskana-core/src/test/java/acceptance/classification/UpdateClassificationAccTest.java

```diff
@@ -276,6 +277,25 @@ public void testUpdateClassificationPrioServiceLevel()
 
     }
 
+    @WithAccessId(
+        userName = "dummy",
+        groupNames = {"businessadmin"})
+    @Test
```

Comment:
@Test(expected=InvalidArgumentException.class) instead of fail ?

Reply:
done